### DISCUSSION TITLE
我看到有多语言模型，请问对应的Dictionary是哪个呢

### DISCUSSION THREAD

**Discussion by BugO0** [qiuqiao/SOFA#13] [General]
![image](https://github.com/qiuqiao/SOFA/assets/54406606/dc194e01-69d8-47bf-9e49-f018f5afbf6e)

**Comment by qiuqiao** [comment]
多语言模型仅用于训练阶段的权重初始化，没法直接用于推理。
事实上，我打算在未来的release中去掉多语言模型的权重。

**Reply by BugO0** [reply]
明白了，感谢解答
